docs/contract_types.rst

```diff
@@ -184,21 +186,33 @@ as they are in different containers:
 Notice that ``medium`` is an optional layer. This means that if it is missing from any of the containers, Import Linter
 won't complain.
 
-This is an example of a contract with sibling layers:
+This is an example of a contract with independent sibling modules:
 
 .. code-block:: ini
 
     [importlinter:contract:my-layers-contract]
-    name = Contract with sibling layers
+    name = Contract with independent sibling modules
```

Comment:
For consistency maybe we should make this `Contract with sibling modules (independent)`?